initializr/src/test/groovy/io/spring/initializr/test/InitializrMetadataTestBuilder.groovy

```diff
@@ -117,7 +117,7 @@ class InitializrMetadataTestBuilder {
 	}
 
 	InitializrMetadataTestBuilder addDefaultLanguages() {
-		addLanguage('java', true).addPackaging('groovy', false)
```

Comment:
I thought that was a mistake in current codebase, and replaced `addPackaging` by `addLanguage` for Groovy, could you please check this @snicoll ?